api/v1/client.go

```diff
@@ -45,9 +45,9 @@ func validateUint64(param string) (uint64, error) {
 // validateBigInt parses a big.Int url parameter.
 func validateBigInt(param string) (*storage.BigInt, error) {
 	i := big.NewInt(0)
-	i, err := i.SetString(param, 10)
-	if err {
-		return nil, common.ErrBadRequest
+	i, ok := i.SetString(param, 10)
```

Comment:
:information_source: https://pkg.go.dev/math/big#Int.SetString ah

Reply:
🤯 my bad; thanks for the fix